### DISCUSSION TITLE
Why isn't GitHub using Discussions for the Community Forum?

### DISCUSSION THREAD

**Discussion by petros** [community/community#23085] [New to GitHub]
<p>Hi folks. I am trying to figure out what platform to use for a new community forum. And I was having GitHub Discussions in mind.</p>
<p>I have just noticed that GitHub themselves are not using GitHub Discussions for their community forum.</p>
<p>I would love more context if possible.</p>
<p>I know Discourse is very popular for community forums. But I also wonder if GitHub Discussions are feature complete enough to stand as a replacement. What would I be missing, for example?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<aside class="quote group-github-employees" data-username="petros" data-post="1" data-topic="260297">
<div class="title">
<div class="quote-controls"></div>
 petros:</div>
<blockquote>
<p>GitHub themselves are not using GitHub Discussions for their community forum</p>
</blockquote>
</aside>
<p>as far as I remember, when this forum started, there was no GitHub Discussions yet, it was just launched in 2020</p>
<p>from the first ever commit of the GitHub Feedback, it was just 2020 too</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181074747-a816aff9-0209-4b53-8a72-81e15bddb20a.png" alt="image" data-base62-sha1="iy9tXT3ZfuJXo6xb4jXb1Nw9bsv" width="535" height="172"></p>
<p>and this forum started earlier than that</p>
<p>as for the forum which is which, I think both will do, unless, say the focus is really GitHub users that will be favorable right there on GitHub</p>
